src/components/header/header.tsx

```diff
@@ -20,57 +20,76 @@ const Header = () => {
       position="sticky"
       id="header"
       top={0}
-      py={{ sm: "xs", md: "s" }}
-      px="s"
       zIndex={999}
       bgColor="ui.white"
+      pt="xs"
     >
       <Box
-        alignItems="center"
         maxWidth="1280px"
-        padding={{ md: "0 15px" }}
         mx="auto"
+        padding="0 16px"
+        sx={{
+          "@media screen and (min-width: 1024px)": {
+            display: "block",
+          },
+          "@media screen and (min-width: 1280px)": {
+            display: "flex",
+            justifyContent: "space-between",
+          },
+          alignItems: "center",
+        }}
       >
         <HStack
+          justify="space-between"
           sx={{
+            "@media screen and (max-width: 768px)": {
+              display: "none",
+            },
+          }}
+        >
+          <DCLogo isMobile={false} />
+          <Box
+            sx={{
+              display: "none",
+              "@media screen and (max-width: 1280px)": {
```

Comment:
Noting another instance of max-width that should be written as min-width if possible.